polaris/utils/types.py

```diff
@@ -128,7 +138,7 @@ class License(BaseModel):
         "https://raw.githubusercontent.com/spdx/license-list-data/main/json/licenses.json"
     )
 
-    id: str
+    id: SupportedLicenseType | str
     reference: Optional[HttpUrlString] = None
 
     @model_validator(mode="after")  # type: ignore
```

Comment:
We no longer need this validator!

We should, however, add a validator to convert the `id` field into the enum if it is a string and to not have to convert between str and enum.

Reply:
Not needed if you decide to replace the enum by a literal type!